graph/telemetry/istio/appender/appender.go

```diff
@@ -34,14 +34,16 @@ func ParseAppenders(o graph.TelemetryOptions) (appenders []graph.Appender, final
 				requestedAppenders[IdleNodeAppenderName] = true
 			case IstioAppenderName:
 				requestedAppenders[IstioAppenderName] = true
+			case MeshCheckAppenderName:
```

Comment:
```suggestion
			case MeshCheckAppenderName:
			case SidecarsCheckAppenderName: // For backward compatibility
```

I know, it's not alphabetical :-)  But, I think it is better to combine the cases that are handled in the same way.  You can leave as-is if you disagree.

Reply:
It is more clear like this